lib/balance-tx/test/spec/Internal/Cardano/Write/Tx/BalanceSpec.hs

```diff
@@ -496,6 +510,9 @@ spec_balanceTransaction = describe "balanceTransaction" $ do
     it "doesn't balance transactions with existing 'returnCollateral'"
         $ property prop_balanceTransactionExistingReturnCollateral
 
+    it "does not balance transactions if no inputs can be created"
```

Comment:
I think this description may read slightly over-simplified, when the test is about when both the utxo set and existing inputs in the tx are empty

Reply:
I agree the description is non-ideal! Though I'm not sure how to improve it and still keep it concise...